### PR TITLE
Make screenreader read login errors

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

**Comment by RoyEJohnson** [comment]
[Bug: Account accessibility issues](https://trello.com/c/GnqWNehB/438-bug-account-accessibility-issues)

File: app/views/layouts/_attention.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% errors = collect_errors %>
 
-<div class="errors">
+<div class="errors" aria-live="polite">
   <%= alert_tag(errors[:alerts])   %>
   <%= notice_tag(errors[:notices]) %>
 </div>
```
Comment (nathanstitt) [inline app/views/layouts/_attention.html.erb]: Hey @RoyEJohnson would `alert` be a better role for these?  I'm not sure but that seems to fit the purpose of the errors div

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Comment (RoyEJohnson) [reply app/views/layouts/_attention.html.erb]: Reading about it, I think so. I'll put that in.

Comment (nathanstitt) [reply app/views/layouts/_attention.html.erb]: Yeah, I'd never used anything other than `polite` before but it seemed appropriate here 😁

File: README.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ link to sign up.
...
+```sh
+$ brew install postgresql
+$ brew services start postgresql
+$ psql postgres
+CREATE ROLE ox_accounts WITH LOGIN;
+ALTER USER ox_accounts WITH SUPERUSER;
+\q
+```
+
 ### Running as a normal Rails app on your machine
```
Comment (nathanstitt) [inline README.md]: Yay!

**Pull request merged**